website_geoengine/models/res_partner.py

```diff
@@ -0,0 +1,117 @@
+from odoo import _, api, fields, models
```

Comment:
Missing a license header

Reply:
I then also added license informations in the JS files